doc/asyncworker.md

```diff
@@ -73,7 +78,7 @@ Definition:
 template<class T>
 class AsyncProgressWorkerBase<T> : public AsyncWorker {
  public:
-  explicit AsyncProgressWorker(Callback *callback_);
+  explicit AsyncProgressWorker(Callback *callback_, const char* resource_name = ...);
```

Comment:
Since you're here: the constructor name is wrong, it should be AsyncProgressWorkerBase.  Ditto the destructor.

Reply:
Done.

Reply:
documentation bug - my bad.  nice catch!